bpf/lib/process.h

```diff
@@ -177,6 +177,7 @@ struct msg_clone_event {
 	struct msg_common common;
 	struct msg_execve_key parent;
 	__u32 pid;
+	__u32 tid;
```

Comment:
I think here pid is always tid, right?
If so, let's just keep `pid`. There is no need to add a new field if it always contains the same information with an existing one.

Reply:
OK, let's go with that.

Reply:
Thank you much appreciated for your time! so I made all the other requested changes